Command/SitemapGeneratorCommand.php

```diff
@@ -56,13 +56,22 @@ class SitemapGeneratorCommand extends ContainerAwareCommand
     protected $output;
 
     /**
+     * @var string
+     */
+    protected $excludeUrlsPattern = null;
+
+    /**
      * SitemapGeneratorCommand constructor.
      *
      * @param RouterInterface $router
      * @param ObjectManager   $objectManager
      * @param array           $config
      */
-    public function __construct(RouterInterface $router, ObjectManager $objectManager, array $config)
+    public function __construct(
```

Comment:
It is better to revert this formatting change and just keep the changes minimal.